package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "loopback-connector-ibmi",
-  "version": "1.0.0-beta.1",
+  "version": "1.0.0",
```

Comment:
Do you have any vision on what the release process is going to look like? For LoopBack modules developed in individual repositories (not loopback-next monorepo), we use `slt-release` from [strong-tools](https://www.npmjs.com/package/strong-tools) package. This tool will automatically bump up the version as part of the release process, and also generate a decent changelog based on commit history.

Unless you have a better tool, I highly recommend to use `slt-release` instead of a manual release process (e.g. based on `npm release`).

Either way, I think we should NOT change the version number until there is a release.